mesonbuild/interpreterbase.py

```diff
@@ -55,6 +55,37 @@ def wrapped(self, node, args, kwargs):
         return f(self, node, args, kwargs)
     return wrapped
 
+#This decorator does the same than permittedKwargs plus:
```

Comment:
__[Flake8]__

block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->